app/utils/getCacheAnalysis.test.ts

```diff
@@ -1,9 +1,277 @@
 import { describe, it, expect } from 'vitest'
 
-import { getTimeToLive } from './getCacheAnalysis'
+import getCacheAnalysis, { getTimeToLive, parseCacheStatus, ServedBySource } from './getCacheAnalysis'
 
 describe('getCacheAnalysis', () => {
-  it.todo('works')
+  it('returns basic cache analysis with minimal headers', () => {
```

Comment:
let's have one test per behaviour, not mega-tests that implicitly verify 18 behaviours. you can replace this and `returns comprehensive cache analysis with all headers present` with focused tests please.

you don't need to comprehensively test the `cacheControl`, `cacheStatus`, and `servedBy` objects because we'll unit test those elsewhere